spec/functional/resource/windows_certificate_spec.rb

```diff
@@ -16,34 +16,39 @@
 #
 
 require "spec_helper"
-require "chef/mixin/powershell_out"
+require "chef/mixin/powershell_exec"
 require "chef/resource/windows_certificate"
 
 module WindowsCertificateHelper
-  include Chef::Mixin::PowershellOut
+  include Chef::Mixin::PowershellExec
 
   def create_store(store)
     path = "Cert:\\LocalMachine\\" + store
     command = <<~EOC
       New-Item -Path #{path}
     EOC
-    powershell_out(command)
+    powershell_exec(command)
   end
 
   def cleanup(store)
     path = "Cert:\\LocalMachine\\" + store
     command = <<~EOC
       Remove-Item -Path #{path} -Recurse
     EOC
-    powershell_out(command)
+    powershell_exec(command)
   end
 
   def no_of_certificates
     path = "Cert:\\LocalMachine\\" + store
+    # Seems weird that we have to call dir twice right?
+    # The powershell cert provider seems to cache the last dir
```

Comment:
I was a bit confused by this comment, I thought this was referring to an infra client provider not realizing "Certifcate Provider" is a Windows thing. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/about/about_certificate_provider?view=powershell-7

Reply:
hoho. yeah I can see that would be confusing. I'll clarify thanks!